src/app/shared/components/template/components/radio-button-grid/radio-button-grid.component.scss

```diff
@@ -77,3 +77,69 @@ $text-color: var(--radio-button-font-color, var(--ion-color-primary));
     }
   }
 }
+
+.radio-buttons[data-variant="flex"] {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-evenly;
+  align-items: center;
+
+  width: 100%;
+  .item {
+    background-color: var(--ion-color-secondary-200);
+    box-shadow: none;
+    border: none;
+
+    width: 140px;
+    height: 140px;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+
+    overflow: hidden;
+    border-radius: 8px;
+    padding: 0px;
+    padding: 12px 14px;
+    margin: 4px 0;
+
+    max-width: calc(7.5rem * 0.9);
+    min-width: calc(6rem * 0.9);
+    max-height: calc(140px * 0.9);
+    min-height: calc(140px * 0.9);
+
+    &[data-selected="true"] {
+      background-color: var(--ion-color-secondary-400);
+      border: 2px solid var(--ion-color-secondary-600);
+      padding: 8px 12px;
+    }
+  }
+  .item-image-container {
+    margin-bottom: 4px;
+    img {
+      max-width: 75px;
+      min-width: 60px;
+      max-height: 52px;
+      object-fit: cover;
+    }
+  }
+  .item-text {
+    width: 100%;
+    display: flex;
+    align-items: flex-end;
+    justify-content: center;
+
+    height: 46px;
+    overflow: hidden;
+    p {
+      color: var(--ion-color-gray-900);
+      font-size: 16px;
```

Comment:
nit(non-blocking): This should read from a variable. We already have a font size variable defined as 16px, `var(--font-size-text-small)`, defined in `src/theme/variables.scss`